drip-woocommerce.php

```diff
@@ -85,3 +85,5 @@ private function product_categories($product_data) {
         return $categories;
     }
 }
+
+include('drip-woocommerce-settings.php');
```

Comment:
I'd put this inside the if clause that checks if Woocommerce is available.

Reply:
Thanks @tdieterich-drip, I attempted to use this: https://docs.woocommerce.com/document/query-whether-woocommerce-is-activated/ but it returns false. Any ideas?

Reply:
I created https://dripcom.atlassian.net/browse/ECI-367 so hopefully we can address this later.